src/measure_theory/measurable_space.lean

```diff
@@ -641,6 +642,12 @@ by simp_rw [measurable_iff_comap_le, measurable_space.pi, measurable_space.comap
 lemma measurable_pi_apply (a : δ) : measurable (λ f : Π a, π a, f a) :=
 measurable.of_comap_le $ le_supr _ a
 
+@[measurability]
+lemma measurable_pi_restrict {δ : Type*} {π : δ → Type*}
+  [hmp : Π a, measurable_space (π a)] (s : set δ) :
+  measurable (@set.pi_restrict δ π s) :=
```

Comment:
Probably better go just state this with `subtype.restrict`

Reply:
Done.